src/Endpoints/Indexes.php

```diff
@@ -219,19 +220,13 @@ public function stats(): array
 
     // Settings - Global
 
-    public function getSettings(): array
+    public function getSettings(): Settings
     {
-        return $this->http->get(self::PATH.'/'.$this->uid.'/settings');
+        return new Settings($this->http->get(self::PATH.'/'.$this->uid.'/settings'));
```

Comment:
I would prefer to get an array instead of an Synonyms object here. This change would break the code of users that expect an array.
We could return the array form like this:

```suggestion
        return (new Settings($this->http->get(self::PATH.'/'.$this->uid.'/settings')))->getIterator()->getArrayCopy();
```

Reply:
Hi @mmachatschek regarding this, Settings being an ArrayAccess class will prevent the code to break, cause the client can loop and use it as an array right?